src/common/_modules/io_proxy/provider.tf

```diff
@@ -0,0 +1,6 @@
+provider "azurerm" {
+  alias           = "prod-cgn"
```

Comment:
Why CGN?

Reply:
my bad, good catch.
I'll change in `prod-platform`

Reply:
Is it a new subscription?

Reply:
mmm, I'm missing something. That value `prod-cgn` come from the `apim` module. What do you think is the right value to use here?

Reply:
It was introduced with [this commit](https://github.com/pagopa/io-infra/commit/0b76c558907baad9b06760b2a82ce739202b03f1), to allow a CGN service to apply changes to APIM. 
I don't think it is the case now, I believe you can remove the entire `provider.tf` file

Reply:
Thanks, fixed in [2d7fa48](https://github.com/pagopa/io-infra/pull/1503/commits/2d7fa488fb821d4af51d74fc2016e02b85fd9627)